### ISSUE TITLE
Many missing textual labels all over the admin UI

### ISSUE THREAD

**Issue by tyrylu** [getgrav/grav-plugin-admin#1736]
When using the grav admin with a screen reader, i noticed many missing labels and other little screen reader annoiances (actions in the page list and very likely somewhere else as well), and also some markup which results in the screen reader ignoring the label (for example the complete admin main menu). I could probably fix these if there is interest, but a question, how is an addition of a translatable string handled? Just add it to the en strings and Crowdin will handle the rest after merge?

**Comment by rhukster** [comment]
There is probably some work required to make Grav admin more screen-reader friendly. Generally, the process is to add the "english" strings to the `languages/en.yaml` in the admin plugin.  Then this file i uploaded to Crowdin to be translated.

**Comment by tyrylu** [comment]
I'll probably look into it, the basics (mainly many, many aria-label attributes) should not be as hard.
